osmnx/speed.py

```diff
@@ -84,12 +87,12 @@ def add_edge_speeds(G, hwy_speeds=None, fallback=None, precision=1):
     # highway type
     for hwy, group in edges.groupby("highway"):
         if hwy not in hwy_speed_avg:
-            hwy_speed_avg.loc[hwy] = group["speed_kph"].mean()
+            hwy_speed_avg.loc[hwy] = agg(group["speed_kph"])
```

Comment:
I am not getting proper behavior here. My highways are filled with nans (I guess as anywhere in the world). But when I use `np.median`, I get result as nan! It seems it is not the same if one use `np.median(series)` or `series.median()`. I am no expert on numpy, but this is what I am observing. Here is simple repro: https://gist.github.com/stalker314314/45ae826371a5b88cbdc76492deeee508

I will try to explore more in coming days!

Reply:
Seems like `Series.median()` function is skipping nans by default (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.median.html?highlight=median#pandas.Series.median), while `numpy.median` does not:) Who could tell:)

Solution that come to my mind could be to either:
1. use `np.nanmedian` (but I would never thought of this unless doc tell me that), or
2. remove nans before calling agg function (but this might limit usability of agg - what if caller want to aggregate with nans!?)

Reply:
And BTW, using `np.nanmedian` does the job and your PR is perfectly correct (if you consider that as a good enough)

Reply:
Thanks for the feedback. Worth noting that as this argument is a function, the user can pass any function they want. They're not constrained to pre-built numpy or pandas aggregations as they can write their own custom function with any special logic they'd like. But I'll add a note to the docstring pointing the user toward `np.median` and `np.nanmedian` as a couple options they might consider.

Reply:
Resolved in f645225